### ISSUE TITLE
Button not showing up in the interface

### ISSUE THREAD

**Issue by nitmir** [jvenant/passff#120]
I'am running firefox 44.0b2. On startup I get the following warning : 
```
1452082952824	addons.manager	WARN	Exception calling callback: TypeError: redeclaration of let PassFF (resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@invicem.pro.xpi!/bootstrap.js -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@invicem.pro.xpi!/modules/common.js:4:1) JS Stack trace: @resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@invicem.pro.xpi!/bootstrap.js -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@common.js:4:1 < startup/</<@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@bootstrap.js:59:5 < startup/<@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/valentin/.mozilla/firefox/6xe2fsl5.default-1438851919914/extensions/passff@bootstrap.js:57:3 < safeCall@AddonManager.jsm:173:5 < makeSafe/<@AddonManager.jsm:189:25 < Handler.prototype.process@Promise-backend.js:934:23 < this.PromiseWalker.walkerLoop@Promise-backend.js:813:7 < this.PromiseWalker.scheduleWalkerLoop/<@Promise-backend.js:747:1
```

the passff button do not show up in the interface and even when I set a shortcut in the settings, nothing happen.

**Comment by nitmir** [comment]
Note that everything works well in firefox 43.0.2

**Comment by jvenant** [comment]
This is a duplicate of #114 and #118.
Fixed on the trunk will be included in 0.1.29
